src/tests/end-to-end/tests/details-view/tabstop.test.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think this is more of a test of the target page visualizations than the details-view, since you have a test case that doesn't involve the details view at all. Maybe `target-page/visualization-tab-stops.test.ts` would be more meaningful than `details-view/tabstop.test.ts`?

Reply:
Moved.